packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -2462,3 +2464,351 @@ Specific property types support a subset of [JSON Schema validations](https://to
 | `number`      | [JSON Number](https://tools.ietf.org/html/rfc7159#section-6) with double-precision 64-bit IEEE 754 floating point number constraint | `minimum` and `maximum`     |
 | `integer`     | [JSON Number](https://tools.ietf.org/html/rfc7159#section-6) with 32-bit signed two's complement integer constraint           | `minimum` and `maximum`     |
 | `array`       | [JSON Array](https://tools.ietf.org/html/rfc7159#section-5)                                                                         |                             |
+
+## Unique attribute
+
+<ApiLifecycle access="ea" />
+
+You can enforce uniqueness for custom properties in the Okta user profile or the Okta group profile, such as employee identification number. You can declare a maximum of 5 unique properties for each user type, and 5 unique properties in the Okta group profile. For unique properties in Okta user profile, the limit of 5 unique properties is applied per-type. You do not need to select the same set of properties for each user type. For example, the 5 custom properties you declare unique for user profile A do not need to match what you declared for user profiles B, C, or D.
+
+Unique properties in the Okta user profile share a single namespace across all [user types](/docs/reference/api/user-types) in an org. If user types A and B both contain a property ice cream and you identify it as unique in both profiles, then if user type A has the value chocolate, no other users of type A or B (or any other user type with ice cream declared unique) can have that value.
+
+Properties that are not unique are not tracked for uniqueness. If the property candy is unique in type E and not unique in type F, and a user of type E has the value caramel for the property, then no other users of type E can have the value caramel for the property, but any number of users of type F can have the value caramel. Although candy is unique in E, it is not unique in F, so the value for the property in users of type F does not matter.
```

Comment:
```suggestion
Properties that aren't unique aren't tracked for uniqueness. If the property `candy` is unique in type E and not unique in type F, and a user of type E has the value `caramel` for the property, then no other users of type E can have the value `caramel` for the property. But any number of users of type F can have the value `caramel`. Although `candy` is unique in E, it isn't unique in F, so the value for the property in users of type F doesn't matter.
```

Reply:
I agree with most of Susan's suggestions, but I don't like starting a sentence with "But" so I would leave that as "comma but" as in the original.  And again, which highlighting form is correct?

Reply:
missed one - should just be `ice cream`

Reply:
i updated in the previous comment - my apologies...thought I caught them all.

Reply:
> I agree with most of Susan's suggestions, but I don't like starting a sentence with "But" so I would leave that as "comma but" as in the original. And again, which highlighting form is correct?

@donwoods-okta - I was trying to split up the sentence as it is incredibly long. Have a suggestion on how we can split that sentence up into two? It's a lot to take in.